isotovideo

```diff
@@ -53,21 +70,24 @@ use Carp 'cluck';
 use Time::HiRes qw(gettimeofday tv_interval sleep time);
 use File::Spec;
 use File::Path;
+Getopt::Long::Configure("no_ignore_case");
 
-# avoid paranoia
-$Getopt::Std::STANDARD_HELP_VERSION = 1;
+my %options;
 
-sub HELP_MESSAGE {
-    print "$0 [-d]\n";
-    print "Parses vars.json and tests the given assets/ISOS\n\n";
-    print " -d enables direct output to STDERR instead of autoinst-log.txt\n";
+sub usage($) {
+    my $r = shift;
+    eval "use Pod::Usage; pod2usage(1);";
```

Comment:
Again perl-critic
`./isotovideo: Expression form of "eval" at line 79, column 5.  See page 161 of PBP.  (Severity: 5)`

Use
```perl
eval { use Pod::Usage; pod2usage(1); };
```
instead